developerSandboxSetup.md

```diff
@@ -31,100 +31,174 @@ There are three options for running a sandbox.
+Three programs are installed: factomd, factom-walletd, and factom-cli. You may also wish to install the GUI [Enterprise Wallet](https://docs.factom.com/#enterprise-wallet).
 
 * Factomd is the main program.  It manages the blockchain, connects to the public network, and enforces the network rules.
-* Fctwallet is an application for holding private keys. It builds Factoid transactions and handles crypto related operations to add user data into Factom.
-* Factom-cli is a program for users to interface with factomd and fctwallet. Through this they create Chains, Entries, and Factoid transactions.
+* factom-walletd is an application for holding private keys. It builds Factoid transactions and handles crypto related operations to add user data into Factom.
+* Factom-cli is a program for users to interface with factomd and factom-walletd. It may be used to create Chains, Entries, and Factoid transactions.
 
 #### Configure Factomd for Sandbox use
 
+Create a folder in your user home folder. The folder should be called `.factom`. If factomd has been run on your computer before this folder will already exist and may already contain part of the factom blockchain.
 
-Create a folder in your user home folder. The folder should be called `.factom`. If factomd has been run on your computer before you may need to first rename the database that is already there: `mv ~/.factom/ldb ~/.factom/ldb_old`.
+In a terminal Linux and Mac: `mkdir -p ~/.factom/m2`or Windows: `mkdir %HOMEDRIVE%%HOMEPATH%\.factom\m2`
 
-In a terminal Linux and Mac: `mkdir ~/.factom`or Windows: `mkdir %HOMEDRIVE%%HOMEPATH%\.factom`
-
-Save the configuration file [factomd.conf](https://raw.githubusercontent.com/FactomProject/FactomCode/master/factomd/factomd.conf) to your `.factom` directory.
+Save the configuration file [factomd.conf](https://raw.githubusercontent.com/FactomProject/factomd/master/factomd.conf) to your `.factom/m2` directory.
 
 * Open `factomd.conf`
+* Change the line `Network` from `MAIN` to `LOCAL`
+	* This will cause factomd to run on its own local network rather than the factom main network.
 * Change the line `NodeMode` from `FULL` to `SERVER`
   * This will make factomd create a blockchain
-* Change `ExchangeRate` to `00000100`
-  * This allows you to create many more Entries with Factoids.  It simulates Factoids having a high market value.
 * Optionally adjust `DirectoryBlockInSeconds`.  600 gives 10 minute blocks, which is more realistic
-  * The default is for 1 minute blocks (60) which is much easier to develop with.  Shorter times have not been tested.
+  * The default is for 6 second blocks which is easier to develop with, but may cuase some strange behavior and errors.
 
 For easier debugging, change the `logLevel` to  `debug`.  This exports pointers to data which is added into factom to the ~/.factom/data/export/ directory.  Adding new entries will add new files to the directory with the specified ChainID.
```

Comment:
I am not sure if we respect this log level still.  the only one I know of is the netdebug level on the command line.